var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -62,7 +62,12 @@ class Perl(Package):
 
     def install(self, spec, prefix):
         configure = Executable('./Configure')
-        configure("-des", "-Dprefix=" + prefix)
+        configure_args = ["-des", "-Dprefix=" + prefix]
+        # Discussion of -fPIC for Intel at:
+        # https://github.com/LLNL/spack/pull/3081
+        if spec.satisfies('%intel'):
+            configure_args.append("-Accflags=" + self.compiler.pic_flag)
+        configure(*configure_args)
```

Comment:
(careful of the 78-char line length, the comment might need to be shortened)

Anyway, this PR looks great and ready to merge!